### DISCUSSION TITLE
[Done ✅] Total balance on wallets page

### DISCUSSION THREAD

**Discussion by britttttk** [btcpayserver/btcpayserver#2306] [Ideas / Feature Requests]
Show the total balance of all the wallets on the wallets page. 

This would be nice on the future dashboard feature.. but can we add it here for now?

**Comment by dstrukt** [comment]
Had a similar thought when stepping through the UI, but don't know if I ever mocked anything up.

ACK, will work it into both (on the design side at least).

**Reply by maxdignan** [reply]
@dstrukt did you happen to get mocks for this? I threw together some code to sum up all of the balances (for each currency), but my UI doesn't look so great 😆 .

**Reply by pavlenex** [reply]
@maxdignan if you wish, you can join us on our design call tomorrow https://github.com/btcpayserver/organization/issues/56 and we can add it to the agenda to discuss about it and provide some feedback on the spot.

**Reply by maxdignan** [reply]
@pavlenex  Thanks for the invite! I'd love to go, but I actually have an interview at that time.

**Reply by pavlenex** [reply]
Alright, you can post what you have right now and we can try to cover it on a meeting and we'll follow up.

**Reply by maxdignan** [reply]
Sounds good! Here is the link to the PR: https://github.com/btcpayserver/btcpayserver/pull/2882 and here is the link on the fork: https://github.com/maxdignan/btcpayserver/pull/1

**Reply by pavlenex** [reply]
Just for the reference @dstrukt 

![Screenshot 2021-09-15 at 18 43 05](https://user-images.githubusercontent.com/36959754/133474757-fb62cb3b-81ae-487d-a2d9-a72923165c7c.png)

**Reply by dstrukt** [reply]
Ahh, thanks, just asked for screenshots on the PR itself .. will drop some screenshots tonight / tomorrow morning, trying to catch up with everything I owe.

**Comment by dstrukt** [comment]
Going to run this by the crew at the design meeting tomorrow morning (PST), but if we're just looking to do the bare bones version + what was aptly called "lurking wife" mode on click and visa versa (imagine we'd save that state, but I leave that to you guys to determine how best to implement), here's the initial stab at it.

You can ignore the other details (been working on the list views and header component, so those made into this mock with the wallet balance additions).

<img width="1433" alt="Screen Shot 2021-09-15 at 10 13 58 PM" src="https://user-images.githubusercontent.com/6250771/133553637-27d3e482-284b-40f2-bb1a-280bb3b4da64.png">
<img width="1415" alt="Screen Shot 2021-09-15 at 10 14 04 PM" src="https://user-images.githubusercontent.com/6250771/133553640-b41ab3da-c987-4f7e-b505-bfc4e16aa0e7.png">

And yes, this will absolutely be on the future dashboard, but for the time being.

Link to Figma page:
https://www.figma.com/file/lE9dYRMqoABkJrn0VdXTE4/?node-id=2%3A9524

All feedback / comments welcome, will follow up with new mocks based on any feedback!
